redbot/cogs/general/general.py

```diff
@@ -267,7 +267,15 @@ async def serverinfo(self, ctx, details: bool = False):
             data.add_field(name=_("Voice Channels"), value=voice_channels)
             data.add_field(name=_("Roles"), value=humanize_number(len(guild.roles)))
             data.add_field(name=_("Owner"), value=str(guild.owner))
-            data.set_footer(text=_("Server ID: ") + str(guild.id))
+            data.set_footer(
+                text=_("Server ID: ")
+                + str(guild.id)
+                + _(
+                    ' | Use "{prefix}serverinfo 1" for more info on the server.'.format(
+                        prefix=ctx.clean_prefix
+                    )
+                )
```

Comment:
1. `.format()` needs to be used *after* the string gets translated (so `_("...").format(...)` rather than `_("...".format(...))`)
2. We're trying to make an effort to not include command names in strings to translate, so using `{command}` replacement field for whole command would be better here.

```suggestion
                + _(' | Use "{command}" for more info on the server.').format(
                    command=f"{ctx.clean_prefix}serverinfo 1"
                )
```